x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_cell_actions.tsx

```diff
@@ -66,19 +64,16 @@ export const getUseCellActionsHook = (tableId: TableId) => {
       if (viewMode === VIEW_SELECTION.eventRenderedView) {
         return undefined;
       }
-      return columns.map((column) => {
-        // TODO use FieldSpec object instead of browserField
-        const browserField: Partial<BrowserField> | undefined = browserFieldsByName[column.id];
-        return {
-          name: column.id,
-          type: browserField?.type ?? '', // When type is an empty string all cell actions are incompatible
-          esTypes: browserField?.esTypes ?? [],
-          aggregatable: browserField?.aggregatable ?? false,
-          searchable: browserField?.searchable ?? false,
-          subType: browserField?.subType,
-        };
-      });
-    }, [browserFieldsByName, columns, viewMode]);
+      return columns.map(
+        (column) =>
+          getFieldSpec(column.id) ?? {
+            name: column.id,
```

Comment:
NIT: 
maybe it would be more consistent to set `name: ''` as we do here:
https://github.com/elastic/kibana/blob/ac4e6b2883de8af175dbf669a72c750a25f16dd1/src/plugins/discover/public/components/discover_grid/discover_grid.tsx#L462